src/main/ml-modules/root/test/test-controller.xqy

```diff
@@ -102,22 +146,29 @@ declare function run-suite($suite as xs:string, $tests as xs:string*, $run-suite
 			attribute passed { fn:count($results/t:test/t:result[@type = 'success']) },
 			attribute failed { fn:count($results/t:test/t:result[@type = 'fail']) },
 			attribute time { functx:total-seconds-from-duration($end-time - $start-time) },
-			$results/*/self::t:test
+			$results/t:test
 		}
 };
 
-declare function run($suite as xs:string, $name as xs:string, $module, $run-teardown as xs:boolean) {
+declare function run(
+		$suite as xs:string,
+		$name as xs:string,
+		$module as xs:string,
+		$run-teardown as xs:boolean,
+		$coverage as map:map?)
+{
 	helper:log(text { "    TEST:", $name }),
 	let $start-time := xdmp:elapsed-time()
 	let $setup := run-setup-or-teardown(fn:true(), $suite)
 	let $result :=
 		try {
 			if (fn:not($setup/@type = "fail")) then
-			(: Avoid returning result of helper:log :)
+			  (: Avoid returning result of helper:log :)
```

Comment:
Please replace the two spaces with a tab to match whitespace in the rest of this file